### DISCUSSION TITLE
Firefly install on RPI is failing on docker-compose

### DISCUSSION THREAD

**Discussion by cbounds89** [firefly-iii/firefly-iii#4951] [Firefly III]
edit: ive fixed this. its amazing what happens when you sleep and relook at the issue. now onto the next issue...

Hey everyone,

I'm new to docker and linux. I was following a tutorial on how to install fireflyiii on a PI.
When I run "sudo docker-compse up -d" I get the follow errors.

`ERROR 1049 (42000): Unknown database '/mysql_scripts/mySqlScript.$'
ERROR: Service 'fireflyiiidb' failed to build: The command '/bin/sh -c /etc/init.d/mysql start && mysql -uroot --password="" -e source /mysql_scripts/mySqlScript.$' returned a non-zero code: 1`

Any help would be appreciated.

Note i've got NextCloud installed on this device not in a container though. (not sure if that running with apache2 is causing any issues)

**Comment by JC5** [comment]
So what was the solution?

**Reply by cbounds89** [reply]
The first issue was the code in the website I used was formatted and I need to change ‘&amp;&amp’ to &&.
The second issue was the link to the source. 
the comments of the site addressed it and he fixed it on GitHub but not the website. 
https://github.com/edwardsj9090/firefly-iii-automation

**Reply by JC5** [reply]
Ah nice, thanks for clarifying!
